From f61b0d891f53faabf1fa766b141a029d1b1994b7 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 11 Jul 2018 13:35:04 +0300
Subject: [PATCH] board: habana: fix abnormal exit value

Since we also output the error string, the script return value in an
abnormal case does not reflect the error.
Add an explicit exit statement to reflect the abnormal execution case.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/remote_load_firmware b/board/habanalabs/common/remote_load_firmware
index 64c4505..b7ed608 100755
--- a/board/habanalabs/common/remote_load_firmware
+++ b/board/habanalabs/common/remote_load_firmware
@@ -72,3 +72,4 @@
 done
 
 echo error
+exit 1;
